homeassistant/components/yale_smart_alarm/manifest.json

```diff
@@ -2,7 +2,8 @@
   "domain": "yale_smart_alarm",
   "name": "Yale Smart Living",
   "documentation": "https://www.home-assistant.io/integrations/yale_smart_alarm",
-  "requirements": ["yalesmartalarmclient==0.1.6"],
-  "codeowners": [],
+  "requirements": ["yalesmartalarmclient==0.3.2","backoff==1.10.0"],
```

Comment:
Backoff is now apart of yalesmartalarmclient so no longer needed to supply it in the manifest